README.md

```diff
@@ -140,6 +128,18 @@ Use your system's environment variables specify an api key and organization to u
 var api = new OpenAIClient(OpenAIAuthentication.LoadFromEnv());
 ```
 
+#### Pass keys directly with constructor (no recommended)
```

Comment:
Changing the name of this section will break the navigation in the table of contents.

Reply:
I'm fine with adding additional text below the section divider to say that this is meant to be used to get started quickly and to test, but should not be recommended for production.

Reply:
Updated the name so it won't break navigation